src/routes/categories/components/CategoriesScrollView.js

```diff
@@ -47,20 +49,20 @@ class CategoriesScrollView extends React.Component<PropsType> {
       return <ScrollView refreshControl={<RefreshControl onRefresh={this.onRefresh} refreshing={loading} />} />
     }
 
-    return (
-      <ScrollView refreshControl={<RefreshControl onRefresh={this.onRefresh} refreshing={loading} />}>
-        <Categories stateView={stateView}
-                    cities={cities}
-                    resourceCache={resourceCache}
-                    language={language}
-                    cityCode={cityCode}
-                    navigation={navigation}
-                    theme={theme}
-                    navigateToCategory={navigateToCategory}
-                    navigateToIntegreatUrl={navigateToIntegreatUrl} />
-      </ScrollView>
-    )
+    return <ScrollView refreshControl={<RefreshControl onRefresh={this.onRefresh} refreshing={loading} />}
+                       contentContainerStyle={{flexGrow: 1}}>
+      <Categories stateView={stateView}
+                  cities={cities}
+                  resourceCache={resourceCache}
+                  language={language}
+                  cityCode={cityCode}
+                  navigation={navigation}
+                  theme={theme}
+                  navigateToCategory={navigateToCategory}
+                  navigateToIntegreatUrl={navigateToIntegreatUrl}
+                  t={t} />
+    </ScrollView>
   }
 }
 
-export default CategoriesScrollView
+export default translate('categories')(CategoriesScrollView)
```

Comment:
Maybe the `t` should rather be passed as prop. Not sure if we have agreed on something though.